packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -79,9 +79,12 @@ const checkboxStyles = css`
   ${boxSizingStyles};
   ${({ size }) => styles[size].baseFontSize};
   line-height: ${({ hasLabel }) => (hasLabel ? lineHeightValue(-1) : "0")};
-  margin: 0 0 ${tokens.space} 0;
   position: relative;
 
+  & + [data-pka-anchor="checkbox"] {
+    margin-top: ${tokens.space};
+  }
```

Comment:
This way a single `<Checkbox>` will not have any margin. But adjacent `<Checkboxes>` will space themselves apart from their preceding sibling.

This is a bit of a compromise with the principle that components should not include margin – "external" styling that interferes with its surroundings. A containing component should be responsible for that kind of layout spacing.  However, especially for the `<Radio>` it's very common to use several of these components together and having the spacing work out of the box is convenient.